tests/libres_tests/res/job_queue/test_job_queue.py

```diff
@@ -224,47 +224,49 @@ def callback(arg):
         job.wait_for()
 
 
-def test_add_ensemble_evaluator_info(tmpdir):
+def test_add_dispatch_info(tmpdir):
     os.chdir(tmpdir)
     job_queue = create_local_queue(SIMPLE_SCRIPT)
     ee_id = "some_id"
-    dispatch_url = "wss://some_url.com"
     cert = "My very nice cert"
     token = "my_super_secret_token"
+    experiment_ingest_uri = "wss://example.org/experiment_ingest"
```

Comment:
If we are removing the `experiment_ingest` entry point in the configuration class, we might as well remove it from this place as well (and just call it `dispatch_url`?) Not a biggy..

Reply:
Just wanted to be clear that we are running the `ExperimentServer` (since providing `experimend_id`), but might remove it I guess - will do.